docs/getstarted/educators-and-students.md

```diff
@@ -0,0 +1,152 @@
+
+## Educator resources
+
+The following resources can help you get started with Visual Studio Code in your classroom:
+
+### Coding and extension packs
+
+It can be frustrating to install all the different software components to get ready to code. There's editors/IDEs, runtimes and compilers, packages or libraries, and different versions of each. We've got you covered with coding and extension packs that can help accelerate your setup.
+
+- Coding Pack for Java: [Windows](https://aka.ms/vscode-java-installer-win) / [macOS](https://aka.ms/vscode-java-installer-mac)
+- [Node.js and JavaScript Education Extension Pack](https://marketplace.visualstudio.com/items?itemName=tanhakabir.node-js-education-extension-pack)
+- [Python Education Extension Pack](https://marketplace.visualstudio.com/items?itemName=tanhakabir.python-education-extension-pack)
+
+### CodeTour
+
+Helping students understand a new codebase can be challenging and time-consuming. Creating descriptive enough instructions can be cumbersome and still not easy enough for students to follow. CodeTour can help you introduce your students to your assignments and projects in a streamlined and interactive way.
+
+[CodeTour](https://marketplace.visualstudio.com/items?itemName=vsls-contrib.codetour) is a Visual Studio Code extension that enables you to create a step-by-step guide through your codebases. This functionality can be helpful in your classroom to break up your assignments into steps for students to follow or to get them familiar with all parts of the assignment by pointing them to relevant pieces within the assignment.
+
+The following short clip shows a student using CodeTour to start working on their app:
+
+![CodeTour in a Space Jam lesson](images/educators-and-students/codetour-space-jam.gif)
+
+Essentially, a "code tour" is a series of interactive steps, each of which are associated with a specific directory, or file/line, and include a description of the respective code. Tours can either be checked into a repo to share with source control, or exported to a "tour file", which enables anyone to replay the same tour, without having to clone any code!
+
+In the following video, an instructor is removing existing instructions embedded in the source code and replacing them with CodeTour steps. Students can then jump from step to step in the large game development project.
+
+![CodeTour in a sample assignment](images/educators-and-students/codetour-example-lesson.gif)
+
+#### Getting started with CodeTour
+
+To start using CodeTour, you need to download the CodeTour extension from the VS Code [Marketplace](https://marketplace.visualstudio.com/vscode).
+
+![CodeTour extension in Extension Marketplace](images/educators-and-students/codetour-extension-marketplace.png)
+
+To use CodeTour effectively in your classroom, we recommend following the guidelines in the [CodeTour repository](https://github.com/microsoft/codetour#getting-started).
+
+## Collaborate with Live Share
```

Comment:
I didn't expect this to be the same level of header as `Educator resources`. If it's a `##` level header because it's fitting for both educators and students (and we don't want to just put it in one), should it be put in its own section like `## General learning resources` or `Tools for both educators and students`, which we could then expand over time to include other tools fitting for both audiences?

Reply:
Good suggestion. I'll add an H2.